src/store/providers/index.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import { FC, ReactNode } from 'react'
+
+import ThemeProvider from './theme-provider'
```

Comment:
use aliases, or explain2me why u use them somewhere, somewhere no.

Reply:
I always forget to check them bcs my code editor doesn't always use them when smth is imported. I'll find out if its is possible to make VS Code always use allias

Reply:
OK, let it be